data/dataset.graphql

```diff
@@ -5,7 +5,7 @@ input DatasetCreateInput {
 }
 
 input WorkspaceSlugAndDatasetSlug {
-  datasetSlug: String!
+  slug: String!
```

Comment:
What motivated you to do this change?
You wanted to remember the good old times of our last PR when you spent 1 hour changing all the inputs of queries from `slug` to `datasetSlug` & `workspaceSlug` 🙃

Reply:
Believe me it was painful 😅 
I did this because the Prisma schema was defined as this:

```
model Dataset {
  id            String       @id @default(uuid()) @db.Uuid
  createdAt     DateTime     @default(now())
  updatedAt     DateTime     @updatedAt
  name          String
  slug          String
  images        Image[]
  labelClasses  LabelClass[]
  workspaceSlug String
  workspace     Workspace    @relation(fields: [workspaceSlug], references: [slug], onDelete: Cascade)

  @@unique(fields: [workspaceSlug, slug], name: "workspaceSlugAndDatasetSlug")
  @@unique(fields: [workspaceSlug, name], name: "workspaceSlugAndDatasetName")
}
```

As you can see the `unique` field relates `workspaceSlug` to `slug` so when I was trying to query a dataset from slugs I would get an error from Prisma saying that the variable `datasetSlug` was not known in the type `workspaceSlugAndDatasetSlug`. The other option would have been to do a `@map` in the Prisma schema from `slug` to `datasetSlug` for Dataset entities, but this would have changed all the query results for dataset entities. For instance, by doing the `@map` we would have had to query a dataset slug like this:

```gql
dataset{
  id
  datasetSlug
}
```

Reply:
Hmm indeed, it feels like the is no good solution... 

Personally I would instead have renames Dataset.slug to Dataset.datasetSlug . But It's not ideal either...